src/main.ts

```diff
@@ -31,6 +31,7 @@ if (process.env.VUE_APP_HAWK_TOKEN) {
   const hawkOptions: HawkInitialSettings = {
     token: process.env.VUE_APP_HAWK_TOKEN,
     release: buildRevision,
+    // vue: Vue,
```

Comment:
?

Reply:
this request is for showing vue addons, not for connecting new catcher. It will be connected at the next request after the new catcher will be published.